content/doc/developer/plugin-governance/deprecating-or-removing-plugin.adoc

```diff
@@ -8,7 +8,7 @@ references:
   title: List of the deprecated Jenkins plugins
 ---
 
-Some plugins may be replaced by others or just become irrelevant (e.g. integration with a service which was shut down).
+Some plugins may be replaced by others or just become irrelevant (e.g. integration with a service which was shut down or a new plugin combining multiple plugins into one).
```

Comment:
```suggestion
Some plugins may be replaced by others or just become irrelevant.
For example, becoming irrelevant can include an integration with a service that was shut down or a new plugin combining multiple plugins into one.
```
Suggestion to remove the usage of `e.g.` based on the [Microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg).

Reply:
Learning something new every day, thanks!

Reply:
... So you replaced it with another term "for example" that Microsoft suggest to avoid cf your link? 😅